components/tabs/tabs.js

```diff
@@ -774,9 +817,26 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(SkeletonMixin(LitElement))
 
 		if (!this._initialized && this._tabs.length === 0) return;
 
-		let selectedTab = this._tabs.find((tab) => tab.selected && tab.state !== 'removing');
+		let selectedTab = null;
```

Comment:
The new code generally follows the same structure as in the default slot change handler, minus the removal.

Reply:
Would we not need `removeTab`, and the `removing` state stuff if the consumer is going to remove by simply not rendering it?

Reply:
Yeah, if we end up getting rid of the `removeTab` functionality and consumers just remove by not rendering then this section would need some updating to be more similar to the default slot change in terms of removal handling as well (minus animation promises)